src/install.ts

```diff
@@ -19,6 +19,7 @@ export async function runCommand(command: string): Promise<string> {
 async function installOnLinux(version: string): Promise<string> {
   const system = runCommand('uname -s')
   const hardware = runCommand('uname -m')
+  version = parseInt(version.split('.')[0], 10) >= 2 ? `v${version}` : version
```

Comment:
Sorry but I found some users [already use `v2.x.x` as version](https://github.com/xhaiker/rust-release.action/blob/d9f674cfe1789e23c5657acd808b1fafb1f8e0b1/.github/workflows/test.yaml#L20-L23), so better to support both `2.x.x` and `v2.x.x`. We can realize it by something like as follows:

```suggestion
  if (!version.startsWith('v') && parseInt(version.split('.')[0], 10) >= 2) {
    version = `v${version}`
  }
```

Please do not forget running `npm run all` before you make a commit :)

Reply:
Yes, that what I meant saying about automatically appending 'v' if needed :)